.github/workflows/publish-docs.yml

```
@@ -68,13 +69,37 @@ jobs:
           mike deploy --push main
 
       - name: Get the release version
-        if: contains(github.ref, 'refs/tags/v') && !github.event.release.prerelease # (don't generate for pre-releases)
+        if: contains(github.ref, 'refs/tags/v')
         id: get_version
         run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
 
-      - name: mike deploy new release
-        if: contains(github.ref, 'refs/tags/v') && !github.event.release.prerelease # (don't generate for pre-releases)
+      - name: Collect tags
+        id: tags
```

The GitHub token needs to be passed explicitly.

```suggestion
        id: tags
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```